### ISSUE TITLE
Some pinned trips are having the same effect as @ing the bot

### ISSUE THREAD

**Issue by DarkBugz** [oldschoolgg/oldschoolbot#5795]
Ive noticed this for two trips so far, farming and guthixiancache

**Comment by TURB042O** [comment]
> Ive noticed this for two trips so far, farming and guthixiancache

yeah l I noticed this beginning of February but I guess I didnt post a issue like I thought I did. only posted in @themrrobert DMs asking how to go about fixing pinned trips.  never got a response. 😅

https://github.com/oldschoolgg/oldschoolbot/assets/133211494/9261b091-814a-4311-991d-6154da11c473

https://github.com/oldschoolgg/oldschoolbot/assets/133211494/3e837e8e-067a-4e39-9c71-1ff6868e13d0
